cinema/serializers.py

```diff
@@ -135,3 +137,40 @@
 
 class OrderListSerializer(OrderSerializer):
     tickets = TicketListSerializer(many=True, read_only=True)
+
+
+class UserRegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, min_length=5)
+
+    class Meta:
+        model = User
+        fields = ["username", "email", "password"]
+
+    def create(self, validated_data):
+        user = User.objects.create_user(**validated_data)
```

Comment:
When creating a user with `create_user`, ensure that the password is hashed. The `create_user` method usually handles this, but it's important to verify that the password is not stored in plain text.